src/DDTrace/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -32,6 +32,13 @@ public function requiresExplicitTraceAnalyticsEnabling(): bool
         return false;
     }
 
+    public function isSymfonyMessengerCommand()
+    {
+        $consoleCommand = $_SERVER['argv'][1] ?? '';
+
+        return $consoleCommand === 'messenger:consume';
+    }
```

Comment:
This may not work: Symfony has an "autocomplete" feature for commands, so `mess:cons` would work too, if it's not ambiguous with other registered commands.

Reply:
Thanks for the info; I wasn't aware of that. That's a bit annoying :p